epictrack-web/src/constants/application-constant.ts

```diff
@@ -96,9 +96,13 @@ export enum SpecialFieldEntityEnum {
   WORK = "WORK",
 }
 
-export const SpecialFields = {
+export const SPECIAL_FIELDS = Object.freeze({

```

Comment:
Just curios, what is the beenfit of doing Object.freeze when its a const?

Reply:
I think it's just good practice. Object.freeze makes the object immutable, it's better practice maybe to use libraries like immutable to deep freeze it.  It's because this is a constant and shouldn't be changed at run time:

In the examples below: SPECIAL_FIELDS is frozen, SPECIAL_FIELDS2 is not frozen:

```
// Attempting to modify a property of the frozen object
try {
  SPECIAL_FIELDS.PROJECT.NAME = "new_name"; // This will throw an error
} catch (error) {
  console.error("Error modifying frozen object:", error.message);
}

// Attempting to add a new property to the frozen object
try {
  SPECIAL_FIELDS.NEW_FIELD = "new_value"; // This will throw an error
} catch (error) {
  console.error("Error adding property to frozen object:", error.message);
}

// Attempting to delete a property from the frozen object
try {
  delete SPECIAL_FIELDS.WORK.RESPONSIBLE_EPD; // This will throw an error
} catch (error) {
  console.error("Error deleting property from frozen object:", error.message);
}

// Modifying a property of the unfrozen object
SPECIAL_FIELDS2.PROJECT.NAME = "new_name"; // This will work

// Adding a new property to the unfrozen object
SPECIAL_FIELDS2.NEW_FIELD = "new_value"; // This will work

// Deleting a property from the unfrozen object
delete SPECIAL_FIELDS2.WORK.RESPONSIBLE_EPD; // This will work
```